blaze-client/src/main/scala/org/http4s/blaze/client/BasicManager.scala

```diff
@@ -15,11 +15,12 @@
  */
 
 package org.http4s
-package client
 package blaze
+package client
 
 import cats.effect._
 import cats.syntax.all._
+import org.http4s.client.{Connection, ConnectionBuilder, RequestKey}
```

Comment:
`Connection` and `ConnectionBuilder` look like things that can be relocated, but that's for another ticket.

Reply:
Sure, Seems like a chore do you have a ticket for that maybe i can also give it a try.